website/integrations/services/apache-guacamole/index.mdx

```diff
@@ -13,29 +13,32 @@ sidebar_label: Apache Guacamole™
+:::
+
 ## Preparation
 
 The following placeholders are used in this guide:
 
 - `guacamole.company` is the FQDN of the Guacamole installation.
 - `authentik.company` is the FQDN of the authentik installation.
 
-Create an OAuth2/OpenID provider with the following parameters:
-
-- **Client Type**: `Confidential`
-- **Redirect URIs**: `https://guacamole.company/` (depending on your Tomcat setup, you might have to add `/guacamole/` if the application runs in a subfolder)
-- **Scopes**: OpenID, Email, and Profile
-
-Under **Advanced protocol settings**, set the following:
+## authentik configuration
 
-- **Token validity**: Any value to configure how long the session should last. Guacamole will not accept any tokens valid longer than 300 Minutes.
-- **Signing Key**: Set the key as `authentik Self-signed Certificate`
+1. From the **authentik Admin interface**, navigate to **Applications** -> **Applications**.
+2. Use the [wizard](https://docs.goauthentik.io/docs/add-secure-apps/applications/manage_apps#add-new-applications) to create a new application and a **OAuth2/OpenID provider**. During this process:
+    - Note the **Client ID**, **Client Secret**, and **slug** values because they will be required later.
+    - Set a `Strict` redirect URI to <kbd>https://<em>guacamole.company</em>/</kbd>. If you have configured [Apache Tomcat](https://tomcat.apache.org/) to run Apache Guacamole in a subfolder, you will need to update this value accordingly.
+    - Select any available signing key.
 
-Note the Client ID value. Create an application, using the provider you've created above.
+## Apache Guacamole Configuration
 
-## Guacamole
+It is recommended to create an admin account in Guacamole before configuring Single Sign-On to simplify the process. Create a user in Guacamole using the same username as in authentik and grant them admin permissions. This step is important to avoid losing access to the Guacamole admin settings, as you may need to revert your changes without it.
 
-It is recommended you configure an admin account in Guacamole before setting up SSO to make things easier. Create a user in Guacamole using the username of your user in authentik and give them admin permissions. Without this, you might lose access to the Guacamole admin settings and have to revert the settings below.
+:::warning
+You can configure Apache Guacamole to use either the `sub` or `preferred_username` as the UID field under `user-name-attribute`. When using `preferred_username` as the user identifier, ensure that the [**Allow users to change username** setting](https://docs.goauthentik.io/docs/sys-mgmt/settings#allow-users-to-change-username) is disabled to prevent authentication issues. The `sub` option uses a unique, stable identifier for the user, while `preferred_username` uses the username configured in authentik.
```

Comment:
Is this a brand new section, or did you move it from elsewhere and I missed that?